Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -27,11 +27,20 @@ final class APIDiffTests: XCTestCase {
         return try SwiftPMProduct.SwiftPackage.execute(args, packagePath: packagePath, env: environment)
     }
 
+    func skipIfApiDigesterUnsupported() throws {
+      guard let tool = try? Resources.default.toolchain.getSwiftAPIDigester() else {
+        throw XCTSkip("swift-api-digester unavailable")
+      }
+      guard localFileSystem.isSymlink(tool) else {
```

Comment:
this seems odd. no better way to tell. cc @abertelrud

Reply:
I agree, this is fairly subtle and won't work in the general case.  Is there a `-version` flag with which it could be invoked and then a more robust check could be made?

Reply:
swift-api-digester is part of the frontend now, so unfortunately there's no version number to check for until "Swift Next" gets an official version number